Jenkinsfile

```diff
@@ -162,18 +162,18 @@ pipeline {
         } // Standard agent tests
 
         stage('Azure Authenticator') {
+          agent { label 'azure-linux' }
```

Comment:
is this the same as `node('azure-linux')`? will this stage run in the azure node?

Reply:
work the same as agent (i verified all tests run)
We cant work with node since it doesnt support post command in case of failure